Building Applications with React and Flux.md

```diff
@@ -0,0 +1,15 @@
+# This is a development environment for React
+
+#### With React comming to Free Code camp soon. I thought this would be a aset to the community.
```

Comment:
```diff
- #### With React comming to Free Code camp soon. I thought this would be a aset to the community.
+ With React comming to Free Code camp soon. I thought this would be an asset to the community.
```